### ISSUE TITLE
Chapter title variations not working

### ISSUE THREAD

**Issue by green345** [JonathanReeve/chapterize#8]
Hi. I tried using chapterize with a text that has chapter titles in this format : '4. The Black Bird' (With number then title), and chapterize returns the headings < 3 error. I used chapterize with a text with titles in the same format except the number was in roman numeral notation followed by a title, and it worked. Not sure why? Any ideas? Thanks for making this program.

**Comment by JonathanReeve** [comment]
Yep, that's because it's a rule-based matching algorithm that only has a few rules. Could you add one for the use case your describing? An accompanying test would also be great.
